presto-jdbc/src/test/java/io/prestosql/jdbc/BaseTestJdbcResultSet.java

```diff
@@ -632,12 +633,12 @@ public void testArray()
         try (ConnectedStatement connectedStatement = newStatement()) {
             checkRepresentation(connectedStatement.getStatement(), "ARRAY[1, 2]", Types.ARRAY, (rs, column) -> {
                 Array array = rs.getArray(column);
-                assertEquals(array.getArray(), new int[] {1, 2});
+                assertThat(array.getArray()).isEqualTo(new int[] {1, 2});
```

Comment:
can you mechanically change other `assertEquals` to `assertThat.equalsTo`?
A thought - if we want such migration maybe we should create a series of `good-first-issue` issues to migrate individual modules.

Reply:
hmm: https://joel-costigliola.github.io/assertj/assertj-core-converting-testng-assertions-to-assertj.html

Reply:
Let's treat this as a follow up, ok? please file appropriate issues, if we want to go into that direction

Reply:
Sure - let's discuss offline if that is direction to go.